pyccel/ast/literals.py

```diff
@@ -222,3 +222,54 @@ def get_default_literal_value(dtype):
     else:
         raise TypeError('Unknown type')
     return value
+
+#------------------------------------------------------------------------------
+
+def convert_to_literal(value, dtype = None, precision = None):
+    """ Convert a python value to a pyccel Literal
+
+    Parameters
+    ----------
+    value     : int/float/complex/bool/str
+                The python value
+    dtype     : DataType
+                The datatype of the python value
+                Default : Matches type of 'value'
+    precision : int
+                The precision of the value in the generated code
+                Default : python precision (see default_precision)
+    """
```

Comment:
Please add `Returns` section to docstring

Reply:
Fixed in e0f925c

Reply:
Thanks